sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/processor/LabelsProcessorFactory.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.sdk.metrics.processor;
+
+public interface LabelsProcessorFactory {
+  static LabelsProcessorFactory noop() {
+    return NoopLabelsProcessor::new;
+  }
+
+  static LabelsProcessorFactory baggageExtractor(BaggageMetricsLabelsExtractor labelsExtractor) {
+    return () -> new BaggageLabelsProcessor(labelsExtractor);
+  }
+
+  /**
+   * Returns a new {@link LabelsProcessorFactory}.
+   *
+   * @return new {@link LabelsProcessorFactory}
+   */
+  LabelsProcessor create();
```

Comment:
I think we should use the same input as for AccumulatorFactory: "resource, libraryinfo, instrumentdescriptor" for consistency. Probably we can be smarter about this and embed them, but for the moment just be consistent.

Reply:
done